libs/designsystem/src/lib/scss/base/_variables.scss

```
@@ -51,6 +51,8 @@ $font-weight: (
   light: 300,
   normal: 400,
   bold: 700,
+  // black will be treated as a named css color if not quoted
+    'black': 900,
```

👍 for commenting why the quote is needed - consider readability though:
```suggestion
    'black': 900, // black will be treated as a named css color if not quoted
```

---

Gotcha! - i will go ahead and create a new issue regarding testing of typography 😀